src/main/scala/cromwell/binding/formatter/SyntaxFormatter.scala

```diff
@@ -133,18 +121,17 @@ class SyntaxFormatter(highlighter: SyntaxHighlighter = NullSyntaxHighlighter) {
   }
 
   private def formatWorkflow(workflow: Workflow): String = {
-    val declarations = workflow.declarations.map{formatDeclaration(_, 1)} match {
-      case x: Seq[String] if !x.isEmpty => s"\n${x.mkString("\n")}\n"
-      case _ => ""
-    }
-    s"""${highlighter.keyword("workflow")} ${highlighter.name(workflow.name)} {$declarations
-        |${workflow.calls.map{formatCall(_, 1)}.mkString("\n")}
+    val declarations = workflow.declarations.map{formatDeclaration(_, 1)}
+    val calls = workflow.calls map {formatCall(_, 1)}
+    val sections = (declarations++calls).filter(_.nonEmpty)
+    s"""${highlighter.keyword("workflow")} ${highlighter.name(workflow.name)} {
+        |${sections.mkString("\n")}
         |}""".stripMargin
   }
 
   private def formatDeclaration(decl: Declaration, level: Int): String = {
```

Comment:
This is one of those examples where I tried both ` map {` as well as `.map(` and neither worked.